Makefile

```diff
@@ -5,90 +5,77 @@ SHELL := bash
 HIDE:=$(if $(VERBOSE),,@)
 UNAME := $(shell uname)
 
-.PHONY: kat rebuild update clean migrate build itest debian-build-image ubuntu-build-image
+.PHONY: $(MAKECMDGOALS)
 
 # Export Docker buildkit options
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 
-kat: env-if-empty clean # This should give you a clean install
-	make build
-	make up
+# Build and bring up all containers (default target)
+kat: env-if-empty build up
```

Comment:
Make does not guarantee that dependencies run sequentially as I found in [this](https://stackoverflow.com/questions/9159960/order-of-processing-components-in-makefile/9160175#9160175) discussion before. In theory this could affect `update` and `reset` as well.

Reply:
Also, the semantics of `make kat` change here since we omit a `clean` now. As this is probably the target that is most used by devs (and perhaps other users), we need to make sure we communicate such a change properly.

Having said that, I do think `make kat` simply meaning "guarantee that I have a running system of the current code" and `make reset` meaning "get me an empty version of the current code" make sense.

Reply:
The default target causing data loss has bit me before, so this was an intended change. Good point about communicating this to the team after merge.

Reply:
> Make does not guarantee that dependencies run sequentially as I found in [this](https://stackoverflow.com/questions/9159960/order-of-processing-components-in-makefile/9160175#9160175) discussion before. In theory this could affect `update` and `reset` as well.

Didn't think of that. Maybe we should consider explicitly disabling parallelism for some targets, as described in https://www.gnu.org/software/make/manual/html_node/Parallel-Disable.html?

Reply:
Seems like a good solution if we can fix this with a one liner!

Reply:
Added `.NOTPARALLEL:`.